compiler/utils/astrepr.nim

```diff
@@ -368,76 +493,77 @@ proc treeRepr*(
 
             else:
               field("typ", conf.textRepr(n.typ))
-              add " != sym.typ" + fgRed
+              add " != sym.typ" + style.err
 
           else:
             field("typ", conf.textRepr(n.typ))
 
-    if not conf.isNil() and trfShowNodeLineInfo in flags and
+    if not conf.isNil() and trfShowNodeLineInfo in rconf and
        n.info != unknownLineInfo:
-      field("info", conf.toMsgFilename(n.info.fileIndex) + fgBlue)
+      field("info", conf.toMsgFilename(n.info.fileIndex) + style.ident)
       add "("
-      add $n.info.line + fgCyan
+      add $n.info.line + style.number
       add "."
-      add $n.info.col + fgCyan
+      add $n.info.col + style.number
       add ")"
 
     case n.kind:
       of nkStrKinds:
         add " "
-        add "\"" & n.strVal + fgYellow & "\""
+        add "\"" & n.strVal + style.strLit & "\""
         addFlags()
+        if hasComment: add "\n"
         addComment()
 
       of nkCharLit .. nkUInt64Lit:
         add " "
-        add $n.intVal + fgBlue
+        add $n.intVal + style.number
         addFlags()
+        if hasComment: add "\n"
         addComment()
 
       of nkFloatLit .. nkFloat128Lit:
         add " "
-        add $n.floatVal + fgMagenta
+        add $n.floatVal + style.floatLit
         addFlags()
+        if hasComment: add "\n"
         addComment()
 
       of nkIdent:
         add " "
-        add n.ident.s + fgGreen
+        add n.ident.s + style.ident
         addFlags()
+        if hasComment: add "\n"
         addComment()
 
       of nkSym:
         add " "
-        add n.sym.name.s + fgCyan
-        hfield "sk", substr($n.sym.kind, 2) + fgBlue
-
-        if not n.sym.owner.isNil() and
-           n.sym.owner.kind in {skModule, skType}:
-          add &"({n.sym.owner.name.s}.{n.sym.name.s})" + termFg(15)
+        add n.sym.name.s + style.ident
+        hfield "sk", substr($n.sym.kind, 2) + style.kind
 
-        add symFields(conf, n.sym, flags, indent)
+        add symFields(conf, n.sym, rconf, indent)
         addFlags()
         addComment()
 
       of nkCommentStmt:
         addFlags()
         add "\n"
+        if hasComment: add "\n"
         addComment()
 
       of nkError:
         if isNil(conf):
-          field("err", substr($n.errorKind(), 4) + termFg(5, 2, 0))
+          field("err", substr($n.errorKind(), 4) + style.errKind)
 
         else:
           let report = conf.getReport(n).semReport
-          field("err", substr($report.kind, 4) + termFg(5, 2, 0))
-          hfield("errid", $n.reportId.int + fgRed)
+          field("err", substr($report.kind, 4) + style.errKind)
+          hfield("errid", $n.reportId.int + style.err)
 
         let (file, line, col) = n.compilerInstInfo()
-        field("info", formatPath(conf, file) + fgBlue)
-        add "(", $line + fgCyan
-        add ".", $col + fgCyan
+        field("info", formatPath(conf, file) + style.ident)
+        add "(", $line + style.number
+        add ".", $col + style.number
```

Comment:
Could we use commas, because I think dots are for tree navigation while commas are for text coordinates, also means from an IDE (thinking vscode extension) this is more expected so it's less churn for now?

All ears for counter arguments.

If it's too much for this PR we can at least agree and leave a note for the future.

Reply:
this was a typo